mobiledgex/edge-cloud-infra
# Added shepherd role to ansible deployments

This is a preliminary pull request to add shepherd service to ansible deployments
   - only added this to staging for now (ansible/staging)
   - added shepherd service role, which is more or less a copy of what crm is with some modified args (ansible/roles/shepherd/tasks/main.yml)
   - added a "--notifySrvAddr" option to crm deployments(ansible/roles/crm/tasks/main.yml)
      - shepherd listens to notifications from crm rather than controller
  - docker network crm will use to send shepherd notifications (ansible/roles/docker/tasks/main.yml)
    - add crm and shepherd containers to that network (ansible/roles/crm/tasks/main.yml)

Questions so far for @venkytv : 
   - How do I test this locally? 
      - with "--check" it fails for some other reasons now for me
   - Is there anything I am missing here?
   - I assume shepherd is already packaged into a mobiledgex docker image, but don't know where to check. Am I correct in my assumption?
